pandas/core/generic.py

```diff
@@ -5589,7 +5589,7 @@ def ftypes(self):
         3    float64:dense
         dtype: object
 
-        >>> pd.SparseDataFrame(arr).ftypes
+        >>> pd.SparseDataFrame(arr).ftypes  # doctest: +SKIP
```

Comment:
change?

Reply:
Sidenote: we should probably deprecate this `ftypes` attribute?

Reply:
Perhaps replace with a `DataFrame.sparse.is_sparse` attribute? Or `DataFrame.sparse.ftypes`?